pkg/pool-utils/contracts/BasePoolAuthorization.sol

```diff
@@ -55,7 +55,9 @@ abstract contract BasePoolAuthorization is Authentication {
         }
     }
 
-    function _isOwnerOnlyAction(bytes32 actionId) internal view virtual returns (bool);
+    function _isOwnerOnlyAction(bytes32) internal view virtual returns (bool) {
+        return false;
+    }
```

Comment:
Note: all these changes were already included in the base of https://github.com/balancer-labs/balancer-v2-monorepo/pull/2059.